tests/test_backend.py

```diff
@@ -3557,29 +3557,29 @@ def func(x):
         self._run_test_case(func, [_OUTPUT], {_INPUT: input_val})
 
     def test_reduce_all(self):
-        input_val = np.random.randint(0, 2, (10, 20)).astype(np.bool)
+        input_val = np.random.randint(0, 2, (2, 20)).astype(np.bool)
```

Comment:
10 isn't great since there's a only a 1/1024 chance of them all matching.  20 sets of size 2 is good.